packages/synapse-react-client/src/components/IconSvg/IconSvg.tsx

```diff
@@ -490,6 +492,8 @@ function IconMapping(props: { icon: string; sx?: SxProps }) {
       return <FlagTwoTone sx={sx} />
     case 'email':
       return <MailOutlineTwoTone sx={sx} />
+    case 'google24':
+      return <GoogleIcon24 sx={sx} />
```

Comment:
Add the Google24 component as a `IconSvg` to simplify icon rendering in LoginMethodButton